src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1455,6 +1468,25 @@ private void ReportIfUnused(LocalSymbol symbol, bool assigned)
             }
         }
 
+        private void ReportUnusedVariables(ImmutableArray<LocalFunctionSymbol> locals)
+        {
+            foreach (var symbol in locals)
+            {
+                ReportIfUnused(symbol, assigned: true);
+            }
+        }
+
+        private void ReportIfUnused(LocalFunctionSymbol symbol, bool assigned)
```

Comment:
The `assigned` parameter seems to always be `true`.  Is there a case where it could be false?  If not should we remove the parameter?